### ISSUE TITLE
Elements must have sufficient color contrast in Dark Theme

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#581]
<dl><dt>Description</dt><dd>The contrast ratio between text and its background is not at least 4.5:1.
In Dark Theme: 'The fifth checkbox is required!' text has insufficient color contrast of 2.25 (foreground color: #fe6d6b, background color: #53636e, font size: 9.0pt (12px), font weight: normal).</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
Small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background.

HOW TO FIX: 
Fix this issue by adjusting the text and/or background to increase the contrast to at least 4.5 to 1.

REFERENCE: 
Deque University: https://dequeuniversity.com/class/visual-design2/contrast/text-against-background

BACKGROUND: 
People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. When the contrast ratio between text and its background is adequate, people who have low vision or are colorblind are more likely to be able to read the text.</dd><dt>Source code</dt><dd><pre><code><div id="id26" class="Error">The fifth checkbox is required!</div></code></pre></dd><dt>Checkpoint</dt><dd>Color Contrast (regular text) (1.4.3.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Checkbox</dd></dl>

**Comment by schne324** [comment]
resolved by #498
